gpt4all-bindings/python/gpt4all/pyllmodel.py

```diff
@@ -327,7 +327,12 @@ def _prompt_callback(token_id):
     # Empty response callback method that just prints response to be collected
     @staticmethod
     def _response_callback(token_id, response):
-        sys.stdout.write(response.decode('utf-8', 'replace'))
+        LLModel.buffer += response
```

Comment:
With the comment above in mind, the problem is then how to access an instance here (it's not possible).

Can't tell you immediately what to do, I'll have to think about a way to get around that bad API design.